src/storage/v2/config.hpp

```diff
@@ -31,12 +31,13 @@ class StorageConfigException : public utils::BasicException {
 /// the storage. This class also defines the default behavior.
 struct Config {
   struct Gc {
-    enum class Type { NONE, PERIODIC };
+    enum class Type { NONE, PERIODIC, PERIODIC_JEMALLOC };
```

Comment:
I would remove PERIODIC_JEMALLOC type. If it is periodic in gc_jemalloc, we will know what is it for.